snc.sh

```diff
@@ -206,6 +206,17 @@ retry ${OC} replace -f pull-secret.yaml
 # Remove the Cluster ID with a empty string.
 retry ${OC} patch clusterversion version -p '{"spec":{"clusterID":""}}' --type merge
 
+# SCP the kubeconfig file to VM
+${SCP} $1/auth/kubeconfig core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/home/core/
```

Comment:
CI is failing because the `SCP` is not defined in the snc.sh side.

Reply:
thanks! done